tests/format/angular/angular/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -9412,20 +9440,26 @@ bindon-ngModel
   should
   be
   {{
-    canSave
-      ? "italic"
-      : "plain"
+    (
+      canSave
```

Comment:
Looks like a regression to have a parenthesis added here

Reply:
Ah yes, good catch. I wonder what could be causing this…

Reply:
ah, no, this is just because this test case has a printWidth of 1. 

I guess we _could_ special-case things to avoid putting parens around single identifiers, but I don't think that's a case worth optimizing for.